### DISCUSSION TITLE
Parse Lavf, lame audio

### DISCUSSION THREAD

**Discussion by rastaman111** [chicio/ID3TagEditor#103] [Q&A] [answered]
Hi
Can this library read files with this encoding?

**Comment by chicio** [comment, marked as answer]
Hi @rastaman111,

the library can read tags from `Data` objects. So if you file respect the id3 standard, you should be able to read them all.

🚀

**Comment by rastaman111** [comment]
Not quite so
If the file has Lavf or lame encoding, then btbltoteka cannot parse them, and you get hieroglyphs

**Reply by chicio** [reply]
My point is: if lame is an encoding on top of mp3 file, you just need to decode it, and then ID3TagEditor will be able to get the tag from the file.
